paddle/fluid/pybind/pybind.cc

```diff
@@ -1348,6 +1349,35 @@ All parameter, weight, gradient are variables in Paddle.
         self.Run(fetch_tensors, fetched_var_name);
       });
 
+  py::class_<framework::Cryption>(m, "Cryption")
+      .def_static(
+          "get_cryptor",
+          []() {
+            std::unique_ptr<Cryption> cryptor =
```

Comment:
just curious, can we just return Cryption::GetCryptorInstance()?

Reply:
it also works, changed to it